0067-quarterly-releases.md

```diff
@@ -0,0 +1,53 @@
+- Feature Name: Release Schedule
+- Start Date: 2022-04-21
+- RFC PR: [apache/tvm-rfcs#67](https://github.com/apache/tvm-rfcs/pull/67)
+
+# Summary
+This RFC proposes that TVM move to a quarterly release schedule. Releases would happen every 3 months or so on a schedule set well in advance, independent of individual feature development in TVM.
+
+# Motivation
+Releases are essential to the usage of TVM, especially now that are beginning to work on publishing binary packages for TVM under PyPi. Making TVM releases frequent forces the release process to become well documented and simple rather than bespoke and only achievable by a small group. Users benefit from releases by seeing that the the project is still under active development and providing an easy way to get new features. As of this RFC it has been five months since the last release. It could easily confuse new users when they expect some TVM feature that was only developed recently but is not present in the latest official release.
+
+
+# Guide-level explanation
+TVM has [release process documentation](https://tvm.apache.org/docs/contribute/release_process.html). This RFC proposes that the release candidate vote thread be abolished in favor of a mechanical schedule where releases happen roughly every three months. A release branch will be cut, evaluated for a period of two weeks, then a release published. Publishing a release entails:
+
+* Gathering and organizing release notes since the last release
+* Posting a source code release on GitHub
+* Uploading the source code to the Apache SVN repository
+* Uploading binary packages to tlcpack
+* Uploading binary packages to PyPi
+
+# Reference-level explanation
+
+Prior to a release a new, lightweight vote would instead be used to nominate a release manager, a committer who will be responsible for guiding along the release process. Releases will roughly match the quarterly [calendar dates](https://en.wikipedia.org/wiki/Calendar_year#Quarters) shifted two weeks earlier (Q1: mid March, Q2: mid June, Q3: mid September, Q4: mid December). The release manager will have final say on all dates and consideration should be given for their personal schedule. The timeline will be as follows:
```

Comment:
Could you motivate why we shift the release two weeks earlier than the calendar quarters?

Reply:
mainly just to avoid conflicting with common holidays (e.g. new year for U.S.), the time interval between is the most important part here so it'd be fine to move the start date to any time if anyone has preferences

Reply:
Is the aim to start a release in this time period or to finish a release? I'm wondering if it's better to aim for the fairly standard January/April/July/October with the flexibility given to the release manager rather than trying to optimise for holidays in a global project. For example, many UK companies reset holidays in April, so there's often a lot of people taking it in mid March.

Reply:
I guess the proposal is to start a release. I do see that we grant flexibility here to the release manager, but I don't know if that's sufficient or not here. It seems like doing an April release 2 weeks later would be ok, if someone was volunteering to be release manager. On the other hand it could hamper participation in the release. Perhaps we can try this a few times and see if we need to make changes?

Reply:
I think the simplest thing is "we aim to deliver a finished release in Jan/Apr/Jul/Oct" as that's straight forward and gives people the flexibility of a month window?